src/storage/QueryBaseProcessor.inl

```diff
@@ -358,6 +361,18 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectVertexProps(
                             const std::vector<PropContext>& props,
                             FilterContext* fcontext,
                             Collector* collector) {
+    if (FLAGS_enable_vertex_cache && vertexCache_ != nullptr) {
+        auto result = vertexCache_->get(std::make_pair(vId, tagId), partId);
+        if (result.ok()) {
+            auto v = std::move(result).value();
+            auto reader = RowReader::getTagPropReader(this->schemaMan_, v, spaceId_, tagId);
+            this->collectProps(reader.get(), "", props, fcontext, collector);
```

Comment:
I'm still not sure what the purpose for empty of key.  😊

Reply:
Because we don't have any properties on key for a vertex.